### ISSUE TITLE
Feature: Pricing Plans Section

### ISSUE THREAD

**Issue by brawblan** [Devslopes-Herd/BaseCamp-LandingPage#14]
## Location
- Home page

## Action
- Using HTML and CSS, build the Pricing Plans section (inside red outline box)

## Acceptance Criteria
- Matches [Figma Design](https://www.figma.com/file/Pucv53KDfj3qvLtRFWMK2E/HTML-CSS-Landing-Page)
- Meet every action item above.
- I have checked everything in the [Definition of Ready](https://github.com/Devslopes-Herd/TeamStandardsAndRules/blob/main/DefinitionOfReady.md) file.
- Submit a Pull Request

![PricingPlans](https://user-images.githubusercontent.com/70041291/184798069-67c4a3c4-432f-4f79-b294-fbe4163161ac.png)

**Comment by j0hannsenn** [comment]
Hey, @brawblan I have read the instructions for contributing, but I am not able to change the status of this issue. So I just wanted to let you now I am currently working on this one. Can you please assign the issue to me?

**Comment by mikesz88** [comment]
> Hey, @brawblan I have read the instructions for contributing, but I am not able to change the status of this issue. So I just wanted to let you now I am currently working on this one. Can you please assign the case to me?

Are you part of the devslopes academy?

**Comment by j0hannsenn** [comment]
No, I am not. Sorry, I didn't noticed, that this is a requirement for contributing.

**Comment by brawblan** [comment]
@j0hannsenn all is well. We appreciate your interest.
